.freeipa-pr-ci.yaml

```diff
@@ -39,206 +43,14 @@ jobs:
         timeout: 3600
         topology: *master_1repl
 
-  fedora-28/caless:
+  fedora-28/test_replica2_with_ca_kra_install:
     requires: [fedora-28/build]
     priority: 50
     job:
       class: RunPytest
       args:
         build_url: '{fedora-28/build_url}'
-        test_suite: test_integration/test_caless.py::TestServerReplicaCALessToCAFull
+        test_suite: test_integration/test_installation.py::TestInstallWithCA1::()::test_replica2_with_ca_kra_install
```

Comment:
The parenthesis in this pytest definition are some hidden trick or is it a typo?

Reply:
I just copy and pasted the test case name. It means that test framework has instantiated the class TestInstallWithCA1 and then executed the case test_replica2_with_ca_kra_install.

PR-CI doesn't like it, because it passes the string through bash without additional quoting. I'm removing it.